wavenet.py

```diff
@@ -211,7 +211,7 @@ def predict_proba(self, waveform, name='wavenet'):
             return tf.reshape(last, [-1])
 
 
-    def loss(self, input_batch, name='wavenet'):
+    def loss(self, input_batch, C=5e-4, name='wavenet'):
```

Comment:
I would add it as a command line parameter. But defaulting it to zero is a good idea